recipes/pendulum/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pendulum" %}
+{% set version = "1.2.4" %}
+{% set sha256 = "1f5554d329e870d114c7c83e30edf5ab1d43250edea2f621fb943811bb3743ba" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  features:
+    - vc9  # [win and py27]
+    - vc14  # [win and py>=35]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - vc 9  # [win and py27]
+    - vc 14  # [win and py>=35]
+  run:
+    - python
+    - python-dateutil
+    - pytzdata
+    - tzlocal
+    - vc 9  # [win and py27]
+    - vc 14  # [win and py>=35]
+
+test:
```

Comment:
With the compiled components, it might be good to run the test suite using pytest.